### ISSUE TITLE
Error install ggtree R 4.2.1 

### ISSUE THREAD

**Issue by bullones** [YuLab-SMU/ggtree#518]
Hello!

My name is Amanda and I'm a PhD Student. I'm trying to install ggtree in RStudio Desktop 2022.07.1+554 with R 4.2.1 and I get an error.
My computer is an Ubuntu 20.04.4.
I've tried to install it both versions:

- Bioconductor version:
BiocManager::install("ggtree")
- github version:
devtools::install_github("YuLab-SMU/ggtree")

But I always get the same error. I hope you can help me. Thank you for your time!

```
> BiocManager::install("ggtree")
'getOption("repos")' replaces Bioconductor standard
repositories, see '?repositories' for details

replacement repositories:
    CRAN: https://cloud.r-project.org

Bioconductor version 3.15 (BiocManager 1.30.18), R 4.2.1
  (2022-06-23)
Installing package(s) 'ggtree'
probando la URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/ggtree_3.4.1.tar.gz'
Content type 'application/x-gzip' length 355778 bytes (347 KB)
==================================================
downloaded 347 KB

* installing *source* package ‘ggtree’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error in get(x, envir = ns, inherits = FALSE) : 
  objeto 'warning_wrap' no encontrado
Error: unable to load R code in package ‘ggtree’
Ejecución interrumpida
ERROR: lazy loading failed for package ‘ggtree’
* removing ‘/home/bullones/R/x86_64-pc-linux-gnu-library/4.2/ggtree’

The downloaded source packages are in
	‘/tmp/RtmpIMoohe/downloaded_packages’
Warning message:
In install.packages(...) :
  installation of package ‘ggtree’ had non-zero exit status
```

**Comment by xiangpin** [comment]
I think your ggplot2 version is the development version (installed from github.). please refer to the same [issue](https://github.com/YuLab-SMU/ggtree/issues/511). You can use `install.package('ggplot2')` to install the released version (ggplot2<=3.3.6). And this issue will be fixed until the next version of `ggplot2` is released, referring to [this](https://github.com/YuLab-SMU/ggtree/pull/512#issuecomment-1178570006).

**Comment by bullones** [comment]
Thank you very much! It fixed my problem! :)
